platform/projects_and_organizations/organization_settings.mdx

```diff
@@ -36,7 +41,10 @@ You can also set the permissions granularly. The **granular permissions are a pa
 
 ## Subscriptions
 
-This page shows your organization's subscription information, including the current plan and usage. You can also change your plan to a different one.
+This page shows your **organization's subscription information**, including the current plan and usage.
+
+You can also **change your plan** to a different one here.
+Plans differ based on the chosen hosting option- you can choose between Tolgee CDN or self hosted. To switch between them them see the Toggle in the centre.

```

Comment:
- not Tolgee CDN but Cloud
- them is written 2 times 
- I don't think "toggle" needs a capital T